packages/guides/tests/unit/Renderer/InventoryRendererTest.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of phpDocumentor.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @link https://phpdoc.org
+ */
+
+namespace phpDocumentor\Guides\Renderer;
+
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @coversDefaultClass \phpDocumentor\Guides\Renderer\InventoryRenderer
+ * @covers ::<private>
+ */
+final class InventoryRendererTest extends TestCase
```

Comment:
This seems to be not implemented. Please remove this class if you are not planning to add unittests :-)

Reply:
woops. must have been a leftover